projects/router/routes.js

```diff
@@ -9,3 +9,14 @@ FlowRouter.route("/projects/add", {
     BlazeLayout.render("mainLayout", {main: "addProject"});
   }
 });
+
+JsonRoutes.add("get", "/projects/:id", function (req, res, next) {
+  var id = req.params.id;
+
+  JsonRoutes.sendResult(res, 200, Projects.findOne(id));
+});
+
+JsonRoutes.add("get", "/projects", function (req, res, next) {
+
+  JsonRoutes.sendResult(res, 200, Projects.find().fetch());
```

Comment:
Please do the `Projects.find().fetch()` in a separate step, and including a semantic variable name such as `allProjects`.